NachoSpigot-API/src/main/java/org/bukkit/entity/Player.java

```diff
@@ -1429,6 +1430,25 @@ public void jump() {
 
     NachoPlayer nacho();
 
+    /**
+     *
+     * @return The player's current amount of absorption hearts.
+     */
+    float getAbsorption();
+
+    /**
+     * Sets the player's current amount of absorption hearts.
+     * @param amount Amount of absorption the player should have.
+     */
+    void setAbsorption(float amount);
+
+    /**
+     * Adds the specified amount of absorption to the player.
+     * @param amount Amount of absorption to add.
+     */
+    void addAbsorption(float amount);
+    // Nacho end
+
```

Comment:
These methods should be moved to HumanEntity

Reply:
Done ✓